.changeset/nasty-hounds-hunt.md

```
@@ -0,0 +1,5 @@
+---
+"@assistant-ui/react": patch
+---
+
+refactor: deprecate UIContentPart
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)